DuckDuckGo/Subscription/Views/PurchaseInProgressView.swift

```diff
@@ -83,10 +83,10 @@ struct ViewHeightKey: PreferenceKey {
     }
 }
 
-struct PurchaseInProgressView_Previews: PreviewProvider {
-    static var previews: some View {
-        PurchaseInProgressView(status: "Completing Purchase... ")
-            .previewLayout(.sizeThatFits)
-            .padding()
-    }
-}
+// struct PurchaseInProgressView_Previews: PreviewProvider {
```

Comment:
Do we need this?   If it gives problems in CI, you can try wrapping in #DEBUG or remove.

Reply:
yes, I removed all the previews because the CI was failing giving a "duplicated preview" error in the unit tests action, I don't think DEBUG will help, I assume the unit tests schema is compiled in debug. I was looking for a IS_CI kind of compilation flag but couldn't find anything.